ocp_resources/ocs_initialization.py

```diff
@@ -0,0 +1,57 @@
+from ocp_resources.constants import TIMEOUT_4MINUTES
+from ocp_resources.resource import NamespacedResource
+
+
+class OCSInitialization(NamespacedResource):
+    """
+    OCSInitialization object.
+
+    API reference (source code linked here due to lack of API doc for this resource):
+        https://github.com/red-hat-storage/ocs-operator/blob/main/api/v1/ocsinitialization_types.go
+
+    Args:
```

Comment:
I missed it, The docstring (`Args` part) should be in `__init__`, please fix.

Reply:
done